pkg/controller/datavolume/controller-base.go

```diff
@@ -631,7 +673,14 @@ func (r *ReconcilerBase) validatePVC(dv *cdiv1.DataVolume, pvc *corev1.Persisten
 	// If the PVC is not controlled by this DataVolume resource, we should log
 	// a warning to the event recorder and return
 	if !metav1.IsControlledBy(pvc, dv) {
-		if pvcIsPopulated(pvc, dv) {
+		requiresNoWork, err := r.pvcRequiresNoWork(pvc, dv)
+		if err != nil {
+			return err
+		}
+		if requiresNoWork {
+			if cc.IsUnbound(pvc) {
+				return fmt.Errorf("unbound populated/adoptable PVC %s/%s", pvc.Namespace, pvc.Name)
```

Comment:
Can we set a condition or emit an event on the datavolume in this case. We want to make this discoverable without looking in the logs.

Reply:
It shouldn't really happen, the webhook makes sure the PVC is bound

Reply:
What about WFFC type storage? It would still be nice if we can bubble up severe issues like this to the user.

Reply:
Yeah I think I have to remove this restriction